docs/conf.py

```diff
@@ -301,4 +301,5 @@ def do_not_plot(*args, **kwargs):
     'user-guide/slicing.html',
```

Comment:
Regarding
```
# Links with images on index page have to point to html files
    # which linkcheck cannot find.
    'user-guide/data-structures.html',
    'user-guide/binned-data/binned-data.html',
    'user-guide/masking.html',
    'visualization/plotting-overview.html',
```
This can be removed now, as we are not linking any more.